CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 0.4.7 - Title - 01 December 2021
+
+New Features:
+- Add Play/Download functions to the Seedbox and various other right pane fixes and additions
+- Add ability of updating the API server urls using DHT
+- Add a link for contributing media information to TMDB
+- Add more languages
+
+Bug Fixes:
+- Fix the Rebuild bookmarks database function
+- Fix the incorrect sorting/filtering of bookmarks when more than 50
+
+Other:
+- Update torrent trackers
+- Update various modules/dependencies
```

Comment:
It's about time that the errors

npm ERR! peer dep missing: backbone@~1.3.3, required by backbone.marionette@3.5.1
npm ERR! peer dep missing: underscore@~1.8.3, required by backbone.marionette@3.5.1
npm ERR! missing: vtt.js@git+https://github.com/gkatsev/vtt.js.git#vjs-v0.12.1, required by video.js@4.12.15
npm ERR! missing: http-node@github:webtorrent/http-node#webtorrent, required by webtorrent@1.5.8

that's shown when running ./make_popcorn.sh gets fixed.
Who's on it?

Reply:
use yarn
vtt - need update video js - https://github.com/popcorn-official/popcorn-desktop/pull/2272 - need css and js fixes
webtorrent updated
and need update marionette